doc/topics/releases/fluorine.rst

```diff
@@ -624,3 +624,14 @@ New style for including from a parent directory:
 .. code-block:: jinja
 
     {% from '../foo' import bar %}
+
+salt-api
+============================
```

Comment:
This is picky, but can you make this line the same length as the text?
```
salt-api
========
```

We just spent a bunch of time fixing up RST syntax errors, so we want to keep these clean :D